docs/api/instruments/keysight/keysightE3631A.rst

```diff
@@ -0,0 +1,12 @@
+##############################################
+Keysight E36312A Triple Output Power Supply
+##############################################
```

Comment:
Did you copy the text from the E3631**2**A?

You will have to cut the "bars" to the length of the text:

```suggestion
##########################################
Keysight E3631A Triple Output Power Supply
##########################################
```